doc/interactivity/CMakeLists.txt

```diff
@@ -37,6 +37,12 @@ install(
   LIBRARY
   DESTINATION
     ${CATKIN_PACKAGE_SHARE_DESTINATION}
+  ARCHIVE
+  DESTINATION
+    ${CATKIN_PACKAGE_SHARE_DESTINATION}
```

Comment:
This should be probably `LIB` resp. `BIN`, shouldn't it?

Reply:
This library is not installed properly and its headers are also not exported.
I would propose to install the library to `CATKIN_PACKAGE_LIB_DESTINATION` (I guess that's an oversight by @mlautman ), but not install the headers at all.
After all we don't want people to depend on a library from the tutorials package.

Reply:
I'm cleaning this up as part of #355 right now.

Reply:
I rebased to get the latest cleanup.